cli/index.js

```diff
@@ -126,6 +127,7 @@ function buildHtmlAndJson({ filePath, output, buildHtml, quiet, titles }) {
     // Find blocks of syntax code and transform it to syntax highlighted code.
```

Comment:
Now this comment no longer makes sense since there are >1 things that happe if the `doc` has a `.body`.